theme/models.py

```diff
@@ -167,7 +167,7 @@ class QuotaMessage(models.Model):
                                                             'or to acquire additional quota, after '
                                                             'which you will no longer be able to '
                                                             'create new resources in HydroShare. ')
-    enforce_content_prepend = models.TextField(default='Your action to add content to HydroShare '
+    enforce_content_prepend = models.TextField(default='Your action '
```

Comment:
It seems any time we change the default text for this model field there will be a new migration file. May be we should override the ```create()``` function of the ```QuotaMessage``` class and pass the default value there for the ```enforce_content_prepend``` field.